### ISSUE TITLE
Can't import DataViewModule

### ISSUE THREAD

**Issue by tuptaker** [primefaces/primeng#5261]
**I'm submitting a ...**  (check one with "x")
```
[ *] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
http://plnkr.co/edit/cT59jz9EpBm69X84ydnU?p=preview

**Current behavior**
"Importing DataViewModule causes error 'has no exported member 'DataViewModule'". On plunkr it manifests as:
GET http://run.plnkr.co/OrSo9gZ7tB9gxioK/@primeng/dataview 404 (Not Found)

**Expected behavior**
DataViewModule can be imported into the application.

**Minimal reproduction of the problem with instructions**
In app.module.ts, import DataViewModule as per primeNG documentation:
import {DataViewModule} from 'primeng/dataview';

See plunker for example:
http://plnkr.co/edit/cT59jz9EpBm69X84ydnU?p=preview

**What is the motivation / use case for changing the behavior?**
I don't want behavior to change, I just want to use the new DataViewModule.

**Please tell us about your environment:**

macos 10.12.6, Microsoft VSC, webpack, local env testing using 'ng serve'.

* **Angular version:** 
5.2.0

* **PrimeNG version:** 
5.2.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
 Typescript 2.5.3

* **Node (for AoT issues):** `node --version` =   
8.5.0

**Comment by cagataycivici** [comment]
Sorry for the confusion, live demo runs on 5.2.1-snapshot, DataView is not in 5.2.0 but in 5.2.1.

**Comment by tuptaker** [comment]
Many thanks, much appreciated!

**Comment by jvdboom** [comment]
How can we install and test 5.2.1?

**Comment by shashidhargm** [comment]
@jvdboom, Follow this blog - https://blog.angular.io/version-5-0-0-of-angular-now-available-37e414935ced
And use 5.2.1 or Whichever you want.
